core/src/test/java/org/apache/druid/data/input/impl/LocalInputSourceTest.java

```diff
@@ -21,6 +21,7 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import nl.jqno.equalsverifier.EqualsVerifier;
+import org.apache.commons.compress.utils.Lists;
```

Comment:
Should this be the one from guava instead? (same for `MaxSizeSplitHintSpecTest`)

Reply:
Oops. Fixed.